src/components/ThemeSetter.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import { useEffect } from 'react';
+
+function setInitialTheme() {
```

Comment:
is this the same with `setColorTheme`? if so we can remove this and import it

Reply:
Removed the duplicate function and imported setColorTheme instead.